src/main.js

```diff
@@ -277,13 +279,79 @@ let load = loader({
     },
   },
 
+  s3Controller: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
+      return new remoteS3.Controller({
+        region: cfg.app.blobArtifactRegion,
+        accessKeyId: cfg.aws.accessKeyId,
+        secretAccessKey: cfg.aws.secretAccessKey,
+      });
+    },
+  },
+
+  // This is a dependency to ease the creation of buckets for, e.g. testing
+  createBuckets: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
+      let s3 = new aws.S3({
+        region: cfg.app.blobArtifactRegion,
+        accessKeyId: cfg.aws.accessKeyId,
+        secretAccessKey: cfg.aws.secretAccessKey,
+      });
+
+      // First, we'll create a public bucket
+      let publicConfig = {
+        Bucket: cfg.app.publicBlobArtifactBucket,
+        ACL: 'public-read',
```

Comment:
I thought you meant on specific objects.  I'll work on this in a bit since it's a little bigger than a quick change.